lib/src/frame.dart

```diff
@@ -237,6 +240,10 @@ class Frame {
           return Frame._parseFirefoxEval(frame);
         }
 
+        if (match[3].contains(' line ')) {
```

Comment:
please delete. seems like this was added accidentally?

Reply:
yes, sorry. removed